pkg/steps/source.go

```diff
@@ -353,7 +353,24 @@ func buildFromSource(jobSpec *api.JobSpec, fromTag, toTag api.PipelineImageStrea
 
 func toEnv(args []api.BuildArg) []corev1.EnvVar {
 	var ret []corev1.EnvVar
-	for _, arg := range args {
+	for i, arg := range args {
+		if arg.ValueFrom != nil {
```

Comment:
We could do 

```
if arg.ValueFrom == nil {
  ret = append ...
  continue
}
```

and switch the rest left, improving readability

Reply:
Done.